### DISCUSSION TITLE
Floppy and HDD Activity Led on PC Emulator

### DISCUSSION THREAD

**Discussion by lgpf** [fdivitto/FabGL#360] [Ideas]
Would be possible to add some kind of activity led for HDD/Floppy on those extra pins? Idea would be create a IBM PC XT and add those details on front panel.

**Comment by linus81** [comment]
Hi, no extra pins are needed. Just add a LED connected between the CS pin of the card reader and the 3.3V power supply.  Connect the anode of the LED to 3.3V with a limiting resistor (for example 220 ohm) and connect the cathode of the LED to the CS pin. The LED turns on when the CS signal is low (I/O activity on the SD card).
![led_sd](https://github.com/user-attachments/assets/12de4401-0452-453d-b900-3a9f8df0fb78)

**Comment by lgpf** [comment]
Thanks. You mean CS at pin 13 that is also available on that header (2, 13,
14, 12, 34, 39, gnd, 3v3)?

Em seg., 23 de set. de 2024, 04:03, linus81 ***@***.***>
escreveu:

**Reply by linus81** [reply]
Hi, exactly. the CS pin is GPIO13 in the MountSD default settings.

SDCard - HSPI
    MOSI  => 17 (12 on TTGO/WROVER)
    MISO  => 16 (2 on TTGO, 35 WROVER)
    CLK   => 14
    **CS    => 13**
